core/src/test/java/io/github/wysohn/triggerreactor/core/main/TriggerReactorCoreTest.java

```diff
@@ -936,6 +936,36 @@ public void command_invTrigger_item() throws Exception {
         assertEquals("STONE", inventoryTriggerManager.get("MyInventory").getItems()[0].getType());
     }
 
+    @Test
+    public void command_invTrigger_canPickupItem() throws Exception {
+        // arrange
+        UUID uuid = UUID.randomUUID();
+        IPlayer sender = mock(IPlayer.class);
+        Injector injector = createInjector();
+        TRGCommandHandler handler = injector.getInstance(TRGCommandHandler.class);
+
+        InventoryTriggerManager inventoryTriggerManager = injector.getInstance(InventoryTriggerManager.class);
+
+        when(sender.getUniqueId()).thenReturn(uuid);
+        when(sender.hasPermission(TRGCommandHandler.PERMISSION)).thenReturn(true);
+        when(pluginManagement.isEnabled()).thenReturn(true);
+
+        // act
+        handler.onCommand(sender, COMMAND_NAME, new String[]{"i", "MyInventory", "create", "54", "#MESSAGE \"Hello World\""});
+
+        // assert
+        assertNotNull(inventoryTriggerManager.get("MyInventory"));
+        assertEquals(54, inventoryTriggerManager.get("MyInventory").getItems().length);
+        assertEquals("#MESSAGE \"Hello World\"", inventoryTriggerManager.get("MyInventory").getScript());
```

Comment:
We can safely remove this parts as it is already tested in [#command_invTrigger_quick()]( https://github.com/TriggerReactor/TriggerReactor/pull/575/commits/251037b2abf94bda620df12dca7d4b9b44d55207#diff-fc2ca3016f00f5a6a916f837db86d4abbec7f44363917ed6b551e000e545de5dR831)